testing/api/test_qc_parallel.py

```diff
@@ -28,6 +24,10 @@ def test_many_qc():
     if multiprocessing.cpu_count() < 2:
         pytest.skip("Can't test parallel behaviour")
 
+    # 'spawn' fix needed to avoid hanging when running tests with coverage
+    # See: https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3661#issuecomment-1026179554
+    multiprocessing.set_start_method('spawn')
```

Comment:
Oops, looks like the tests are failing because this method has already been called elsewhere. 

```
Traceback (most recent call last):
  File "/home/runner/work/spinalcordtoolbox/spinalcordtoolbox/testing/api/test_qc_parallel.py", line 29, in test_many_qc
    multiprocessing.set_start_method('spawn')
  File "/home/runner/work/spinalcordtoolbox/spinalcordtoolbox/python/envs/venv_sct/lib/python3.7/multiprocessing/context.py", line 242, in set_start_method
    raise RuntimeError('context has already been set')
RuntimeError: context has already been set
```

I'll debug this a little more then re-request once I've figured it out.

Reply:
Fixed by 17afec6, I think!